src/Network/IMAP.hs

```diff
@@ -317,13 +319,28 @@ store conn sequenceSet dataItem flagList = do
                                      encodeUtf8 dataItem, flagsToText flagList]
   sendCommand conn command
 
+uidStore :: (MonadPlus m, MonadIO m, Universe m) => IMAPConnection ->
+  T.Text -> T.Text -> [Flag] -> m CommandResult
+uidStore conn sequenceSet dataItem flagList = do
+  let command = BSC.intercalate " " ["UID STORE", encodeUtf8 sequenceSet,
+                                     encodeUtf8 dataItem, flagsToText flagList]
+  liftIO $ putStrLn $ show command
```

Comment:
we don't need that

Reply:
I agree - sorry about that

Reply:
no worries, just remove it and I'll merge the PR :)